module/meta.go

```diff
@@ -42,6 +43,22 @@ func (pr ProviderRequirements) Equals(reqs ProviderRequirements) bool {
 	return true
 }
 
+type CloudBackend struct {
+	Data backend.CloudData
+}
+
+func (be *CloudBackend) Equals(b *CloudBackend) bool {
+	if be == nil && b == nil {
+		return true
+	}
+
+	if be == nil || b == nil {
+		return false
+	}
+
+	return be.Data.Equals(b.Data)
+}
+
```

Comment:
Given that `backend` is a public package, we shouldn't need an extra struct here, just to wrap it?
```suggestion
```